test/JDBC/expected/linked_servers-vu-verify.out

```diff
@@ -54,3 +54,39 @@ mssql_server2#!#only_user_no_password
 mssql_server3#!#<NULL>
 ~~END~~
 
+
+-- Try to drop a linked server login that does not exist (should throw error)
+EXEC sp_droplinkedsrvlogin @rmtsrvname = "invalid_server", @locallogin = NULL
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: server "invalid_server" does not exist)~~
+
+
+-- Try to drop a linked server login with @rmtsrvname = NULL (should throw error)
+EXEC sp_droplinkedsrvlogin @rmtsrvname = NULL, @locallogin = NULL
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: @servername cannot be NULL)~~
+
+
+-- Try to drop a linked server login with locallogin != NULL (should throw error saying that only localogin = NULL is supported)
+EXEC sp_droplinkedsrvlogin @rmtsrvname = "mssql_server", @locallogin = "login_1"
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Only @locallogin = NULL is supported)~~
+
+
+-- drop all the linked server logins that have been created
+EXEC sp_droplinkedsrvlogin @rmtsrvname = "mssql_server2", @locallogin = NULL
+GO
+
+EXEC sp_droplinkedsrvlogin @rmtsrvname = "mssql_server3", @locallogin = NULL
+GO
+
+-- Call sp_droplinkedsrvlogin from master.dbo schema
+EXEC master.dbo.sp_droplinkedsrvlogin @rmtsrvname = "mssql_server", @locallogin = NULL
```

Comment:
After doing these drops we should query sys.linked_logins to check if the logins were actually dropped or not